diesel/src/query_builder/mod.rs

```diff
@@ -31,6 +31,7 @@ mod select_statement;
 mod sql_query;
 mod update_statement;
 pub(crate) mod upsert;
+pub use upsert::on_conflict_target_decorations::DecoratableTarget;
```

Comment:
Minor nit:

It would be good to match the style of other `pub use` statements and do it like this:

```rust
pub use self::upset::on_conflict_target_decorations::DecoratableTarget;
```

Also would be good to be moved down to the other `use` statements, as well.